src/test/java/com/mavenit/cucumber/PageObject/ResultsPage.java

```diff
@@ -101,10 +105,7 @@ public void selectTypeFilter(String typeOption) {
     }
 
     public void selectCapacityFilter(String capacityOption) {
-        /* need to know how to implement show more
-        * */
-       // new WebDriverWait(driver, 10).until(ExpectedConditions.elementToBeClickable(By.cssSelector(".Accordionstyles__ButtonLink-pegw6j-3.bRQRVq")));
-        //driver.findElement(By.cssSelector("button[data-test='component-accordion-button-show-more']")).click();
+        driver.findElement(By.cssSelector("div[data-facet='backpack capacity (l)']>.Accordionstyles__ButtonLink-pegw6j-3.bRQRVq")).click();
```

Comment:
Locator too long, can it be shorter

Reply:
I cant find any other locator for show more. So, I connected child with its parent.